### ISSUE TITLE
Plotting the anchor boxes

### ISSUE THREAD

**Issue by corentin87** [Jumabek/darknet_scripts#21]
In visualise_anchors.py, the width and height of the anchors boxes are converted back to the image size (416x416) by multiplying by 32
```
w=int (w*stride)
h=int(h*stride)
```
However when plotting the rectangle you are using (w,h) as your bottom-right corner. Shouldn't it be
```
x = 10+i*stride_w
y = 10+i*stride_h
cv2.rectangle(blank_image, (x, y) ,(x+w, x+h), colors[i])
```
?

**Comment by Jumabek** [comment]
Hi @corentin87 , 
You are absolutely correct, good catch!
Fixed accordingly.

**Comment by Jumabek** [comment]
Yes. I also did not see any improvement. Sometimes it was even worse. However. Since you are detecting only person you can use anchors for other purpose. But lets day you use only one anchor, then custom anchor surely outperform the default ones

**Comment by corentin87** [comment]
what do you mean by "_you can use anchors for other purpose_" ?
